paddle/phi/backends/onednn/onednn_reuse.h

```diff
@@ -1267,16 +1336,18 @@ class BatchNormOneDNNHandler
                                 dnnl::batch_normalization_backward>(engine,
                                                                     cpu_place) {
     // Flags are added by bitwise OR operation
-    auto flags = dnnl::normalization_flags::use_scale_shift;  // 001
+    auto flags = dnnl::normalization_flags::use_scale |
+                 dnnl::normalization_flags::use_shift;  // 001
     if (global_stats)
       flags |= dnnl::normalization_flags::use_global_stats;  // 010
     if (fuse_with_relu && test_mode)
       flags |= dnnl::normalization_flags::fuse_norm_relu;  // 100
```

Comment:
Why do we need these numbers here?
```suggestion
    auto flags = dnnl::normalization_flags::use_scale |
                 dnnl::normalization_flags::use_shift;
    if (global_stats)
      flags |= dnnl::normalization_flags::use_global_stats;
    if (fuse_with_relu && test_mode)
      flags |= dnnl::normalization_flags::fuse_norm_relu;
```

Reply:
removed